packages/browser/src/eventbuilder.ts

```diff
@@ -265,13 +265,11 @@ export function eventFromUnknownInput(
  */
 export function eventFromString(
   stackParser: StackParser,
-  input: string,
+  input: string & { __sentry_template_string__?: string; __sentry_template_values__?: string[] },
```

Comment:
Can we extract `ParemeterizedString` from this PR as a type export it from @sentry/types? Then we can reuse it here.

Reply:
Tried to replace it everywhere where I think it makes sense. Thanks for the tip!